one_fm/api/api.py

```diff
@@ -150,32 +127,75 @@ def push_notification(employee_id, title, body):
     # See documentation on defining a message payload.
     for registration_token in registration_tokens:
         message = messaging.Message(
-            data= {
-            "title": title,
-            "body": body,
-            "showButtonCheckIn": 'True',
-            "showButtonCheckOut": 'True',
-            "showButtonArrivingLate": 'True'
-            },
-            android=messaging.AndroidConfig(
-                notification=messaging.AndroidNotification(
-                    title=title,
-                    body=body,
-                    click_action = "oneFmNotificationCategory2",
+                data= {
+                "title": title,
+                "body": body,
+                "showButtonCheckIn": 'True',
+                "showButtonCheckOut": 'True',
+                "showButtonArrivingLate": 'True'
+                },
+                android=messaging.AndroidConfig(
+                    notification=messaging.AndroidNotification(
+                        title=title,
+                        body=body,
+                        click_action = "oneFmNotificationCategory1",
+                    ),
                 ),
-            ),
-            apns=messaging.APNSConfig(
-                payload=messaging.APNSPayload(
-                    aps=messaging.Aps(
-                        badge=0,
-                        mutable_content= 1
+                apns=messaging.APNSConfig(
+                    payload=messaging.APNSPayload(
+                        aps=messaging.Aps(
+                            badge=0,
+                            mutable_content= 1,
+                            category = "oneFmNotificationCategory1"
+                        ),
                     ),
                 ),
-            ),
-            token=registration_token,
+                token=registration_token,
             )
         response = messaging.send(message)
-        print(response)
     return response
     # See the BatchResponse reference documentation
-    # for the contents of response.
\ No newline at end of file
+    # for the contents of response.
+
+# This function is used to send notification through Firebase CLoud Message. 
+# It is a rest API that sends request to "https://fcm.googleapis.com/fcm/send"
+# Params: employee_id e.g. HR_EMP_00001
+@frappe.whitelist()
+def push_notification_rest_api(employee_id):

```

Comment:
This function is sending a push notification message to FCM for a checkin reminder. Please name the function and provide its description appropriately.